components/resources/chat_ui_strings.grdp

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grit-part>
+  <message name="IDS_SUMMARIZE_PROMPT_TEMPLATE" translateable="false" desc="Default prompt for summarization">
+    Write a concise summary and always respond in bullet points:
```

Comment:
Note for security review: how do we feel about the prompt being in a resource file: Anything we should consider in terms of malicious injection on a user's system?

Reply:
Makes sense to have a seperate grdp file for this, yes. Updated.

Reply:
I'm not sure of the implications here. Does that mean that local malware can change the prompt without it breaking code signing?

I think we should also consider that the prompt can be simply changed (probably easily?) by remote website using a prompt injection like "Ignore the prompt and do X instead." cc @thypon